valentyusb/usbcore/cpu/eptri.py

```diff
@@ -18,42 +18,90 @@
 from ..sm.transfer import UsbTransfer
 from .usbwishbonebridge import USBWishboneBridge
 
-# Register Interface:
-#
-# pullup_out_read: Read the status of the USB "FS" pullup.
-# pullup_out_write: Write the USB "FS" pullup state
-#
-# SETUP - Responding to a SETUP packet from the host
-# setup_read: Read the contents of the last SETUP transaction
-# setup_ack: Write a "1" here to advance the data_read fifo
-# setup_empty: "0" if there is no SETUP data.
-# setup_epno: The endpoint the SETUP packet was destined for
-#
-# EPOUT - Data from the host to this device
-# epout_data_read: Read the contents of the last transaction on the EP0
-# epout_data_ack: Write a "1" here to advance the data_read fifo
-# epout_last_tok: Bits 2 and 3 of the last token, from the following table:
-#    USB_PID_OUT   = 0
-#    USB_PID_SOF   = 1
-#    USB_PID_IN    = 2
-#    USB_PID_SETUP = 3
-# epout_epno: Which endpoint contained the last data
-# epout_queued: A response is queued and has yet to be acknowledged by the host
-#
-# EPIN - Requests from the host to read data from this device
-# epin_data_write: Write 8 bits to the EP0 queue
-# epin_data_empty: Return 1 if the queue is empty
-# epin_epno: Which endpoint the data is for.  You must write this byte to indicate data is ready to be sent.
-# epin_queued: A response is queued and has yet to be acknowledged by the host
-#
-# ep_stall: a 32-bit field representing endpoitns to respond with STALL.
-
-
-### Handle SETUP packets
-###
-### SETUP packets must always respond with ACK.  Sometimes, they are followed
-### by DATA packets, but not always.
+"""
```

Comment:
This documentation block is probably wrong, and will need to get updated before we ship.  Is it supposed to go above or below the `class` definition line?

Reply:
```python
#/usr/bin/env python

"""
Module docstring
"""

import os
import sys

def f(blah):
   """ Function docstring. """"
   return blah + 2


class B:
   """ Class docstring. """

  def __init__(self, a, b):
     """ Constructor docstring. 

     (Frequently only one of the class and __init__ doc string is used) """
     pass

  def a(self):
     """ Method docstring. """"
     pass

```